conf/modules/short_variant_calling.config

```diff
@@ -29,7 +29,8 @@ process {
         ext.args = { [
             "--sample_name=${meta.id}",
             "--model_type=${params.deepvariant_model_type}",
-            '-vcf_stats_report=False'
+            '-vcf_stats_report=False',
+            '--haploid_contigs chrX,chrY'
```

Comment:
I think you need a sex check here, as for a female the X chromomsome will be diploid. 
Something like:
`( ${meta.sex} == 1 ) ? '--haploid_contigs="chrX,chrY"' : ''`

Reply:
Yes, of course 🤦